deps/build-deps-linux.sh

```diff
@@ -46,6 +46,8 @@ make install
 
 mkdir -p $BASEDIR/libpng
 tar -xzf $SRCDIR/libpng-1.6.29.tar.gz -C $BASEDIR/libpng --strip-components 1
+cd $BASEDIR/libpng
+patch -Np1 -i $SRCDIR/libpng-1.6.29-apng.patch
```

Comment:
Is this already part of the `libpng-1.6.29.tar.gz` source in https://github.com/discord/lilliput-dep-source ?

Reply:
The distribution of libpng-apng is a bit weird - the APNG spec hasn't been officially ratified by whatever committee is in charge of PNG, so support for it can't be included in libpng. Instead, libpng-apng is distributed as a [patch](https://sourceforge.net/projects/libpng-apng/) applied to the base libpng library. The version of libpng currently being used doesn't have this patch included, so it needs to be applied separately.